### PR TITLE
cleanup question marks in CDK code

### PR DESCRIPTION
**Pull request by stephane-airbyte**
just some kotlin cleanup

### CODE REVIEW

**Comment by stephane-airbyte** [comment]
* **#37518** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/37518?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stephane-airbyte and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: airbyte-cdk/java/airbyte-cdk/datastore-bigquery/src/main/kotlin/io/airbyte/cdk/db/bigquery/TempBigQueryJoolDatabaseImpl.kt [hunk 1]
```diff
@@ -14,21 +14,20 @@ import org.jooq.exception.DataAccessException
...
 
 /** This class is a temporary and will be removed as part of the issue @TODO #4547 */
-class TempBigQueryJoolDatabaseImpl(projectId: String?, jsonCreds: String?) : Database(null) {
-    val realDatabase: BigQueryDatabase
-
-    init {
-        realDatabase = createBigQueryDatabase(projectId, jsonCreds)
-    }
+class TempBigQueryJoolDatabaseImpl(
+    projectId: String?,
```
Comment (edgao) [inline airbyte-cdk/java/airbyte-cdk/datastore-bigquery/src/main/kotlin/io/airbyte/cdk/db/bigquery/TempBigQueryJoolDatabaseImpl.kt:18]: doublechecking: these are intentionally still nullable?

File: airbyte-cdk/java/airbyte-cdk/datastore-bigquery/src/main/kotlin/io/airbyte/cdk/db/bigquery/TempBigQueryJoolDatabaseImpl.kt
Comment (edgao) [inline airbyte-cdk/java/airbyte-cdk/datastore-bigquery/src/main/kotlin/io/airbyte/cdk/db/bigquery/TempBigQueryJoolDatabaseImpl.kt:1]: TIL this class exists....?

**Review by edgao — APPROVED** [review]
what's up with changing private -> protected in a few places?

**Review by gisripa — APPROVED** [review]

File: airbyte-cdk/java/airbyte-cdk/datastore-bigquery/src/main/kotlin/io/airbyte/cdk/db/bigquery/TempBigQueryJoolDatabaseImpl.kt (hunk 1 above)
Comment (stephane-airbyte) [reply airbyte-cdk/java/airbyte-cdk/datastore-bigquery/src/main/kotlin/io/airbyte/cdk/db/bigquery/TempBigQueryJoolDatabaseImpl.kt:18]: yeah, this was about removing the trivial ones. Removing all of them is going to be a whole lot harder...

**Comment by stephane-airbyte** [comment]
### Merge activity

* **May 23, 1:53 PM EDT**: @stephane-airbyte started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/37518).

**Pull request merged**